libraries/WDT/src/WDT.cpp

```diff
@@ -0,0 +1,72 @@
+#include "WDT.h"
+
+// Constructor
+APM3_WDT::APM3_WDT()
+{
+    // The watchdog only runs off of the LFRC
+    am_hal_clkgen_control(AM_HAL_CLKGEN_CONTROL_LFRC_START, 0);
+    am_hal_wdt_init(&watchdogConfig);
+}
+
+// Configure the watchdog timer
+void APM3_WDT::configure(uint8_t clock, uint8_t interrupt, uint8_t reset)
+{
+  WDT->CFG_b.CLKSEL = clock; 
+  WDT->CFG_b.INTVAL = interrupt; 
+  WDT->CFG_b.RESVAL = reset; 
+}
+
+// Set the LFRC clock frequency of the watchdog timer
+void APM3_WDT::setClock(uint8_t clock)
+{
+  WDT->CFG_b.CLKSEL = clock; 
+}
+
```

Comment:
List the options in comments?  Ie.
`//Options are:
// AM_HAL_WDT_LFRC_CLK_DEFAULT (128Hz)
// AM_HAL_WDT_LFRC_CLK_128HZ
// AM_HAL_WDT_LFRC_CLK_16HZ
// AM_HAL_WDT_LFRC_CLK_1HZ
// AM_HAL_WDT_LFRC_CLK_1_16HZ
// AM_HAL_WDT_LFRC_CLK_OFF`